Palindrome.java

```diff
@@ -7,11 +7,11 @@ public static void main(String[] args) {
         String rev = "";
```

Comment:
@bhaveshrmahajan can we remove this extra variable?

Reply:
yes sure

Reply:
problem is solve pls check it